[2025-02-24T04:55:09.982+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-24T04:55:10.002+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.great_func scheduled__2025-02-23T00:00:00+00:00 [queued]>
[2025-02-24T04:55:10.013+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.great_func scheduled__2025-02-23T00:00:00+00:00 [queued]>
[2025-02-24T04:55:10.014+0000] {taskinstance.py:2867} INFO - Starting attempt 2 of 7
[2025-02-24T04:55:10.026+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): great_func> on 2025-02-23 00:00:00+00:00
[2025-02-24T04:55:10.033+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1506) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-24T04:55:10.032+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'great_func', 'scheduled__2025-02-23T00:00:00+00:00', '--job-id', '122', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmplcezsvmc']
[2025-02-24T04:55:10.035+0000] {standard_task_runner.py:72} INFO - Started process 1507 to run task
[2025-02-24T04:55:10.035+0000] {standard_task_runner.py:105} INFO - Job 122: Subtask great_func
[2025-02-24T04:55:10.076+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.great_func scheduled__2025-02-23T00:00:00+00:00 [running]> on host 79e0cb3ed452
[2025-02-24T04:55:10.154+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='great_func' AIRFLOW_CTX_EXECUTION_DATE='2025-02-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-23T00:00:00+00:00'
[2025-02-24T04:55:10.155+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-24T04:55:10.180+0000] {logging_mixin.py:190} INFO - *** chào Linh Thùy, i am 1 days old!!!
[2025-02-24T04:55:10.181+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-24T04:55:10.189+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-24T04:55:10.190+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=great_func, run_id=scheduled__2025-02-23T00:00:00+00:00, execution_date=20250223T000000, start_date=20250224T045510, end_date=20250224T045510
[2025-02-24T04:55:10.251+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-24T04:55:10.277+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-24T04:55:10.280+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
